broadinstitute/single_cell_portal_core
# Browse collections functionality (SCP-3595, SCP-3596)

SCP-3595, SCP-3596. This includes some work from a [now-closed PR](https://github.com/broadinstitute/single_cell_portal_core/pull/1129), but the functionality has changed enough that I thought it warranted a new PR.  This replaces the COVID-19 button with a 'Browse Collections' button that goes to the Collections list page I demoed previously.  The COVID-19 space is now listed there as a 'collection'.  

![image](https://user-images.githubusercontent.com/2800795/131899543-56b52de0-5a13-494a-918a-a2598aa2c3fb.png)


![image](https://user-images.githubusercontent.com/2800795/131898201-fd425a33-ed19-4bc6-9190-7b23a9fb0f6c.png)

This removes the Covid-19 feature flag as it is no longer necessary

Note this work does not actually create a Covid 'collection.', that work is ticketed as SCP-3633.   See also SCP-3634 and SCP-3635 for related work. 

TO TEST:
 1. Go to home page, confirm "Browse collections" button appears
 2. Click it, and confirm it takes you to collections list page
 3. Confirm collections list page shows the covid-19 page, and any other collections that the current user has access to
 4. Confirm links on the collections page take you to the desired places
 5. Sign in as an admin
 6. Confirm you now see all collections in the directory
 7. Use the edit page to toggle a collection published/unpublished, and confirm it saves, and the directory is updated accordingly
